apps/src/storage/dataBrowser/TableControls.jsx

```diff
@@ -67,7 +67,7 @@ class TableControls extends React.Component {
         </div>{' '}
         <div style={styles.buttonWrapper}>
           {experiments.isEnabled(experiments.APPLAB_DATASETS) && (
-            <VisualizerModal />
+            <VisualizerModal key={this.props.tableName} />
```

Comment:
Interesting!  I hadn't seen `key` used this way before, so I did some digging and learned that this is the ["Fully uncontrolled component with a `key`"](https://reactjs.org/blog/2018/06/07/you-probably-dont-need-derived-state.html#recommendation-fully-uncontrolled-component-with-a-key) pattern, recommended [by the React docs](https://reactjs.org/docs/react-component.html#static-getderivedstatefromprops) when "you want to 'reset' some state when a prop changes."  Great find!